python/lsst/ts/scheduler/scheduler_csc.py

```diff
@@ -459,6 +459,13 @@ async def do_resume(self, data):
                 result="Resuming Scheduler operation.",
             )
 
+            if (
+                self._tasks["target_production_task"] is not None
+                and self._tasks["target_production_task"].done()
+            ):
```

Comment:
What if there is no `target_production_task` key in `self._tasks`? Should it be started in that case as well or can that be ignored as is the case now?